backend/app/util.py

```diff
@@ -1,4 +1,6 @@
 import asyncio
+import sys
+import traceback
```

Comment:
Am I missing something - do we need these here? Also, do we not have a "imports are not used"-pre-commit step? :D